Documentation/design-docs/tiered-compilation-guide.md

```diff
@@ -0,0 +1,46 @@
+# Tiered Compilation Guide
+
+# Introduction
+
+.NET Core and ASP.NET ship mostly as [ReadyToRun](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/readytorun-overview.md) (R2R) for better startup performance, but steady-state performance is lower compared to only using the JIT due to some R2R versioning constraints.
```

Comment:
It looks like 2.2 preview 2 has already released with these release notes: https://blogs.msdn.microsoft.com/dotnet/2018/09/12/announcing-net-core-2-2-preview-2/

I don't completely agree that all of what you suggested should be in the release notes, but I think at least there should be a link to something like this, which can be a good starting point to the various resources about tiering.

Given the short runway I had to finish this, I am trying to finish this but (c) above looks like it would take more time. Running MusicStore seems to be rather more complicated than it should be, and taking a step back, I don't think that would generally be very useful to people. More useful would be the subset of that content that shows them how to compare tiering vs no-tiering with their own app (a generic project), which I would rather head towards for the 2.2 fork of the demo.

That said, I also don't think (c) is super important. The most important aspects of what is covered here I believe are - the known issue that is most likely to occur (I'm open to extending that to add a link to other known issues), and how to provide feedback.

Before I spend too much time on forking the demo, I'd prefer to get something out that could be linked from the release notes soon, considering that the release notes are already live. There are more pressing issues than forking the demo at the moment.

Reply:
I agree that forking the demo isn't pressing.
And if getting something out there fast appears more valuable than waiting to get feedback feel free to do that.